signup-page.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<style>
+    .main-container{
+        background-color: #EDF0F5;
+        padding: 1% 3%;
+        margin: 3% 35%;
+        font-size: 18px;
+        font-family:Georgia, 'Times New Roman', Times, serif;
```

Comment:
Why one is with inverted comma, others are not?

Reply:
Here is another font name is only Times

Reply:
My question to you was, `Times New Roman` is inside '' while `Georgia`, `Times` and `serif` not?